File: VisitBosnia/visit_bosnia_mobile/lib/model/appUser/app_user.dart
class AppUser {
  int? id;
  String? firstName;
  String? lastName;
  String? userName;
  String? image;
  String? phone;
  String? email;
  bool? isBlocked;

  AppUser(
      {this.id,
      this.firstName,
      this.lastName,
      this.userName,
      this.image,
      this.phone,
      this.email,
      this.isBlocked});

  AppUser.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    firstName = json['firstName'];
    lastName = json['lastName'];
    userName = json['userName'];
    image = json['image'];
    phone = json['phone'];
    email = json['email'];
    isBlocked = json['isBlocked'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['firstName'] = this.firstName;
    data['lastName'] = this.lastName;
    data['userName'] = this.userName;
    data['image'] = this.image;
    data['phone'] = this.phone;
    data['email'] = this.email;
    data['isBlocked'] = this.isBlocked;

    return data;
  }
}

File: VisitBosnia/VisitBosnia.Services/Database/AppUser.cs
using System;
using System.Collections.Generic;

namespace VisitBosnia.Services.Database
{
    public partial class AppUser
    {
        public AppUser()
        {
            AgencyMembers = new HashSet<AgencyMember>();
            AppUserFavourites = new HashSet<AppUserFavourite>();
            AppUserRoles = new HashSet<AppUserRole>();
            EventOrders = new HashSet<EventOrder>();
            PostReplies = new HashSet<PostReply>();
            Posts = new HashSet<Post>();
            Reviews = new HashSet<Review>();
        }

        public int Id { get; set; }
        public string FirstName { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string Email { get; set; } = null!;
        public string? Phone { get; set; }
        public string UserName { get; set; } = null!;
        public string PasswordHash { get; set; } = null!;
        public string PasswordSalt { get; set; } = null!;
        public byte[]? Image { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public bool IsBlocked { get; set; }

        public virtual ICollection<AgencyMember> AgencyMembers { get; set; }
        public virtual ICollection<AppUserFavourite> AppUserFavourites { get; set; }
        public virtual ICollection<AppUserRole> AppUserRoles { get; set; }
        public virtual ICollection<EventOrder> EventOrders { get; set; }
        public virtual ICollection<PostReply> PostReplies { get; set; }
        public virtual ICollection<Post> Posts { get; set; }
        public virtual ICollection<Review> Reviews { get; set; }
    }
}
